csunplugged/config/settings/testing.py

```diff
@@ -71,3 +71,15 @@
     "test_without_migrations",
     "dev.apps.DevConfig",
 ]
+
+# Save production LANGUAGES setting in case any tests need to refer to it
+PRODUCTION_LANGUAGES = LANGUAGES  # noqa: F405
+
+# Override production value of LANGUAGES - this is what django-modeltranslation
+# will use when adding translated fields to the models on startup, which is
+# necessary to test i18n backend functionality.
+LANGUAGES = (
```

Comment:
Yes, it really would be ideal to set this on a per-test basis - this is what I tried to achieve initially. Unfortunately, a change in the available languages requires a change to the database schema because modeltranslation has to add/remove database columns for each language for a translatable field.

Overriding on a per-test-class basis works for `test_TranslatableModelLoader.py` because it never touches the database, although some trickery was still needed in the `setUpClass` method to get modeltranslation to pick up the new settings (see 8ceadf61bda145a2afe4025a17e300626605c298). That being said, despite being possible, I'm not currently overriding the LANGUAGES setting in that test class.

All the other loader tests are actually loading models into the db, and so would require schema modifications for each test case. This actually is possible - the unit tests for the `django-modeltranslation` package do exactly this - but it's messy (see https://github.com/deschler/django-modeltranslation/blob/master/modeltranslation/tests/tests.py#L110) and I believe this approach to be overkill for our purposes.

Setting LANGUAGES in the testing.py settings file was the best/simplest compromise I could come up with, but I'm open to other suggestions.

Reply:
After reading all that, I definitely agree!